src/sql/sql-server/create.sql

```diff
@@ -0,0 +1,183 @@
+
+-- THIS SCRIPT MUST BE MODIFIED TO FOLLOW THE NAMING CONVENTIONS HERE: https://ripplelabs.atlassian.net/wiki/display/DT/Database+design+patterns
+-- THE PARENT DIRECTORY SHOULD ALSO BE RENAMED TO THE KNEX CLIENT USED TO CONNECT TO SQL SERVER
+
+--
+-- Sequences 
+--
+
+CREATE SEQUENCE seq_n_notifications_pk AS bigint
+START WITH 1
+INCREMENT BY 1
+NO CYCLE
+CACHE 100
+go
+
+CREATE SEQUENCE seq_n_cases_pk AS bigint
+START WITH 1
+INCREMENT BY 1
+NO CYCLE
+CACHE 100
+go
+
+--
+-- Table
+--
+
+CREATE TABLE N_LU_CASE_STATUS
+( 
+  CASE_STATUS_ID       bigint  NOT NULL ,
+  CASE_STATUS_CODE     varchar(20)  NOT NULL ,
+  CASE_STATUS_DESC     varchar(255)  NULL ,
+  DB_CREATED_DTTM      datetime  NOT NULL 
+     DEFAULT  getdate(),
+  DB_UPDATED_DTTM      datetime  NOT NULL 
+     DEFAULT  getdate(),
+  DB_UPDATED_USER      varchar(40)  NOT NULL 
+       DEFAULT CURRENT_USER
+)
+go
+
+CREATE UNIQUE CLUSTERED INDEX XPKN_LU_CASE_STATE ON N_LU_CASE_STATUS
```

Comment:
I haven't change the sql server scripts to conform to the new standards, since we don't yet support sql server. I left a comment explaining this at the top of the file. I added them for completeness, but I can remove them if it's confusing.